admin/src/scenes/classe/view.tsx

```diff
@@ -67,7 +67,7 @@ export default function View() {
 
   const getClasse = async () => {
     try {
-      const { ok, code, data: classe } = await api.get(`/cle/classe/${id}`);
+      const { ok, code, data: classe } = await api.get(`/cle/classe/${id}`, { withPopulate: true });
```

Comment:
```suggestion
      const { ok, code, data: classe } = await api.get(`/cle/classe/${id}`);
```
le populate était pour le service, il n'est pas nécessaire ici.

sinon pour aller plus en profondeur : 
- "pupulate" est un mot propre à l'implementation du back il vaut mieux rester agnostique.
- sinon je mettrais "1" au lieu de "true" pour éviter de faire une comparaison de string dans le controller `=== "true"`
- si l'ancien comportement par défaut était withDetail=1 alors ne pas préciser le parametre devrait faire la meme chose qu'avant.